samples/nrf9160/nrf_cloud_agps/Kconfig

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "nRF Cloud and A-GPS Sample Settings"
+
+config CLOUD_BACKEND
+	string "String that selects the cloud backend to be used"
+	default "NRF_CLOUD"
+
+config CLOUD_MESSAGE
+	string "Custom message published periodically to cloud"
+	default "{\"state\":{\"reported\":{\"message\":\"Hello Internet of Things!\"}}}"
+
+config POWER_SAVING_MODE_ENABLE
+	bool "Request PSM from cellular network"
```

Comment:
Most boolean options enable things. It is best practice to rename to POWER_SAVING_MODE (or perhaps add a prefix to disambiguate).

CLOUD_POWER_SAVING_MODE

Reply:
Agreed, I'll rename to `LTE_PSM`, `LTE_POWER_SAVING_MODE` or something along those lines to specify what sort of power saving this enables